packages/ledger-amino/src/ledgersigner.ts

```diff
@@ -10,16 +10,16 @@ import {
 import { HdPath } from "@cosmjs/crypto";
 import Transport from "@ledgerhq/hw-transport";
 
-import { AddressAndPubkey, LaunchpadLedger, LaunchpadLedgerOptions } from "./launchpadledger";
+import { AddressAndPubkey, LedgerConnector, LedgerConnectorOptions } from "./ledgerconnector";
 
 export class LedgerSigner implements OfflineAminoSigner {
-  private readonly ledger: LaunchpadLedger;
+  private readonly ledger: LedgerConnector;
```

Comment:
Let's rename this to better distinguisn the (Ledger)Signer and (Ledger)Connector

```suggestion
  private readonly connector: LedgerConnector;
```